### PR TITLE
add config to disable escape html

### PR DESCRIPTION
**Pull request by hylin**
#65

### CODE REVIEW

**Comment by cwtuan** [comment]
@hylin 
Thanks for the contribution.
Please also add tying in https://github.com/alibaba/react-intl-universal/blob/master/typings/index.d.ts

**Comment by jhaenchen** [comment]
@hylin you'll want to agree to the SLA

**Comment by cwtuan** [comment]
@hylin Did you sign SLA already?

**Comment by ls-endurance** [comment]
I need this change. how do we speed this up?

File: src/index.js [hunk 1]
```diff
@@ -39,7 +39,8 @@ class ReactIntlUniversal {
       urlLocaleKey: null, // URL's query Key to determine locale. Example: if URL=http://localhost?lang=en-US, then set it 'lang'
       cookieLocaleKey: null, // Cookie's Key to determine locale. Example: if cookie=lang:en-US, then set it 'lang'
       locales: {}, // app locale data like {"en-US":{"key1":"value1"},"zh-CN":{"key1":"值1"}}
-      warningHandler: console.warn // ability to accumulate missing messages using third party services like Sentry
+      warningHandler: console.warn, // ability to accumulate missing messages using third party services like Sentry
+      escapeHtml: true // disable escape html in variable mode
```
Comment (jhaenchen) [inline src/index.js]: Surely this should default to `false`?
```suggestion
      escapeHtml: false // disable escape html in variable mode
```

**Comment by ls-endurance** [comment]
ill get right on it. thank you

**Pull request merged**